agile-development/stand-ups/readme.md

```diff
@@ -1,110 +1,110 @@
+
+- 0-1 min = Excellent
+- 1-3 min = Great
+- 3-5 min = Good
+- 5+ min = Needs Improvement
+
+### New Tasks Created After Stand-up
+
+How many tasks are being generated after the stand-up that didn't exist before? This can indicate how much unplanned work is being done. If creating new tasks after stand-up becomes routine (especially for the same story), this could indicate the story is at higher risk of not being completed. It could also indicate a shift in focus to an unplanned objective.
+
+## Facilitation Guidance
+
+The process lead should facilitate the stand-up meeting.
+
+### Speak to Tasks
+
+When answering what was worked on and what will be worked on, refer directly to tasks. This has two benefits:
+
+1. The answer will naturally be short (e.g. "I finished task 114 yesterday; which was to update the build of the api container image. I will be starting on task 115 today; which is to update the release pipeline for the same container image.").
+2. Unplanned work will be easily identifiable. If the person is unable to refer to a task, that typically indicates they are working on something unplanned or out of scope for the sprint (e.g. Yesterday, I was attempting to optimize some of the unit tests from last sprint to run faster).
+
+If a contributor is not working on an existing sprint task they need to either create a new task under an existing sprint story to reflect that work, or defer that work until its scheduled for a later sprint.
+
+**If a contributor provides an update without referring to a task, ask the contributor which task.**
+
+### Parking Lot Discussion Items
+
+As contributors are answering the three questions, if another contributor has a question or issue to share, they should reserve until after all contributors have answered finished. Once each member has answered all three questions, the process lead should open up the floor to anyone who may have an open question or unresolved issue to share. This portion of the ceremony is often referred to as the "Parking Lot".
+
+**Parking lot discussions are optional for participants**. Ensure discussion leaders call out necessary parties for their discussion points upfront, allowing those not needed to leave the meeting.
+
+### Start On Time
+
+Make a best effort to begin answering the 3 questions as close to the scheduled start time as possible. Try not to waste time upfront on chit-chat or waiting on all team members to join. This can extend the meeting time significantly. Starting immediately will help ensure stand-ups remain effective and useful over time.
+
+### Same Time Everyday
+
+Stand-up should be held at the same time each day. The meeting time should be mutually agreed-upon by the contributors, and should take into consideration time zones, working schedules, and other factors so that every team member can reasonably participate.
+
+### Schedule Stand-ups for Mid-Morning
+
+```plaintext
```

Comment:
Can we get rid of this coffee cup?

Reply:
For context, I just moved the existing text, but happy to remove the coffee cup :)